app/assets/stylesheets/components/_slider.scss

```diff
@@ -0,0 +1,88 @@
+$track-w: 16em;
+$track-h: .625em;
+$track-bg: $grey-4;
+$track-fill: linear-gradient(90deg, $grey-2 25%, $grey-2 100%);
+
+$thumb-d: 1.5em;
+
+
+// MIXINS
+@mixin track($flag: true) {
+  width: $track-w;
+  height: $track-h;
+  border-radius: $track-h / 2;
+  box-shadow: inset 0 1px .125em $grey-1;
+  background: $track-bg;
+
+  @if $flag {
+    background: $track-fill no-repeat $track-bg;
+    background-size: 0 100%;
+  }
+}
+
+@mixin thumb() {
+  width: $thumb-d;
+  height: $thumb-d;
+  border-radius: $thumb-d;
+  border: 3px solid $color-option-leave-pot-untouched;
```

Comment:
We shouldn't be using an option specific colour here. If the slider changes when used in the context of an option we should modify the colour in context.